backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/utils/ApplicationHandler.kt

```diff
@@ -88,16 +89,16 @@ class ApplicationHandler(
     }
 
     fun isValidPreVerifiedApplication(): Boolean {
-        val isAlreadyVerifiedSet =
-            application.applicationDetails.blueCardEntitlement?.workAtOrganizationsEntitlement?.list?.any {
-                it.isAlreadyVerified == true
-            } ?: false
-        if (isAlreadyVerifiedSet) {
-            // check if api token is set and valid, if not throw unauthorized exception
```

Comment:
this comment shouldn't be remobed since its not implemented here

Reply:
You are right, but as the other PR is already open and based on this PR, i think it is not worth the effort to now readd this comment, rebase the other PR and then remove it there.